test/TestHelper.jl

```diff
@@ -0,0 +1,84 @@
+#=
+    TestHelper
+
+This module defines helper functions, objects, and constants to be used by
+various files in the test folder.
+=#
+module TestHelper
+
+using ClimaCore: ClimaCore, Geometry, Meshes, Domains, Topologies, Spaces, Fields, InputOutput
+using ClimaComms
+using NCDatasets
+
+export create_space, gen_ncdata!
+
+"""
+    create_space(FT; comms_ctx = ClimaComms.SingletonCommsContext(), 
+        R = FT(6371e3), ne = 4, polynomial_degree = 3)
+
+Initialize a space on a sphere with the given parameters.
+Used for debugging and testing.
+
+# Arguments
+- FT: [DataType] Float type
+- comms_ctx: [ClimaComms.AbstractCommsContext] context used for this operation.
+- `R`: [FT] radius of the sphere underlying space.
+- `ne`: [Integer] number of elements used in the space's mesh.
+- `polynomial_degree`: [Integer] degree of the polynomial used to represent
+    the space (number of GLL nodes - 1).
+"""
+function create_space(FT; comms_ctx = ClimaComms.SingletonCommsContext(), R = FT(6371e3), ne = 4, polynomial_degree = 3)
+    domain = Domains.SphereDomain(R)
+    mesh = Meshes.EquiangularCubedSphere(domain, ne)
+
+    if comms_ctx isa ClimaComms.SingletonCommsContext
+        topology = Topologies.Topology2D(mesh, Topologies.spacefillingcurve(mesh))
+    else
+        topology = Topologies.DistributedTopology2D(comms_ctx, mesh, Topologies.spacefillingcurve(mesh))
+    end
+
+    Nq = polynomial_degree + 1
+    quad = Spaces.Quadratures.GLL{Nq}()
+    space = Spaces.SpectralElementSpace2D(topology, quad)
+    return space
+end
+
+"""
+    gen_ncdata(FT, path, varname, val)
+
+Create an NCDataset with lat/lon dimensions containing the value `val` for the
+variable `varname`, and store it at `path`.
+
+# Arguments
+- `FT`: [DataType] Float type.
+- `path`: [String] location to store output datafile.
+- `varname`: [Symbol] variable name.
+- `val`: [FT] value to store as `varname` at all indices.
+"""
+function gen_ncdata!(FT, path, varname, val)
```

Comment:
The bang is not consistent with the docstring. AFAIK the convention is only to use bangs when arguments are mutated, so I would remove it here.